src/util/portable-script.js

```diff
@@ -36,11 +36,20 @@ async function makePortableProxyScriptUnix(
   const sourcePath = path.isAbsolute(source) ? source : `$(dirname "$0")/../${source}`;
 
   await fs.mkdirp(destination);
-  await fs.writeFile(
-    filePath,
-    `#!/bin/sh\n\n${environment}"${sourcePath}"${prependedArguments} "$@"${appendedArguments}\n`,
-  );
-  await fs.chmod(filePath, 0o755);
+
+  if (process.platform === 'win32') {
+    await fs.writeFile(
+      filePath + '.cmd',
+      `@${environment}"${sourcePath}" ${prependedArguments} ${appendedArguments} %*\r\n`,
```

Comment:
I've merged approaches from linux branch of this `if` with what cmd-shim does (that's where `%*` is coming from). But I'd need @arcanis to double check I'm not doing anything wrong here. Thanks.